lib/facade_converter.ts

```diff
@@ -130,19 +130,23 @@ export class FacadeConverter extends base.TranspilerBase {
     this.emit(ident);
   }
 
-  private getHandler<T>(symbol: ts.Symbol, m: ts.Map<ts.Map<T>>): T {
-    var {fileName, qname} = this.getFileAndName(symbol);
+  private getHandler<T>(n: ts.Node, symbol: ts.Symbol, m: ts.Map<ts.Map<T>>): T {
+    var {fileName, qname} = this.getFileAndName(n, symbol);
     var fileSubs = m[fileName];
     if (!fileSubs) return null;
     return fileSubs[qname];
   }
 
-  private getFileAndName(symbol: ts.Symbol): {fileName: string, qname: string} {
+  private getFileAndName(n: ts.Node, symbol: ts.Symbol): {fileName: string, qname: string} {
     while (symbol.flags & ts.SymbolFlags.Alias) symbol = this.tc.getAliasedSymbol(symbol);
     let decl = symbol.valueDeclaration;
     if (!decl) {
       // In the case of a pure declaration with no assignment, there is no value declared.
       // Just grab the first declaration, hoping it is declared once.
+      if (!symbol.declarations || symbol.declarations.length === 0) {
+        this.reportError(n, 'no declarations for symbol');
```

Comment:
Done. This will just complain that it cannot find declarations for the given symbol that was encountered at this location. I think that's fair - this shouldn't really happen in the first place (symbol with no declaration doesn't make sense).